examples/server.mjs

```diff
@@ -0,0 +1,44 @@
+import express from 'express';
+
+const app = express();
+const port = 3000;
+
+import * as dotenv from "dotenv";
+dotenv.config()
+
+import { load } from "@azure/app-configuration-provider";
+const connectionString = process.env.APPCONFIG_CONNECTION_STRING;
+const appConfig = await load(connectionString, {
+    featureFlagOptions: {
+        enabled: true,
```

Comment:
what's this `enabled` used for?

Reply:
https://github.com/Azure/AppConfiguration-JavaScriptProvider/blob/main/src/featureManagement/FeatureFlagOptions.ts#L15